include/frg/array.hpp

```diff
@@ -128,6 +128,18 @@ constexpr auto array_concat(const Ts &...arrays) {
 
 namespace std {
 
+template <class T, size_t N>
+constexpr size_t size(const T (&array)[N]) noexcept
+{
```

Comment:
This brace should be on the line above, follow the other functions in this file. Same goes for the function below.